src/components/NewProject.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="new-project">
+    <h1>New Project</h1>
```

Comment:
Minor consistency:
Register Cluster vs New Project.
This might be a signal to investigate some form inheritance of these .vue files. Maybe we can avoid duplication of the style definition for example.
Take this as food for thought, not to be done in this PR